internal/service/cluster/resource_cluster_schema_migration_test.go

```diff
@@ -0,0 +1,131 @@
+package cluster_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/cluster"
+)
+
+func TestAccClusterRSClusterMigrateState_empty_advancedConfig(t *testing.T) {
```

Comment:
i think we're using "migration" here in a different way as in the typical migration tests.   in fact for instance these tests execute in acc not in mig GH actions because the test names

not a strong opinion but don't know if calling the files "migration" is confusing, maybe i would put these tests together with the regular test file.

Reply:
I think we can rename these to `resource_cluster_state_upgrader_test` to be more specific, does that address your concerns?

Reply:
updated to `resource_cluster_state_upgrader_test` let me know in case of concerns

Reply:
removing "migrate/migration" from the name works for me, thanks!